appinventor/appengine/src/com/google/appinventor/client/TopToolbar.java

```diff
@@ -186,10 +185,10 @@ public void updateMenuState(int numSelectedProjects, int numProjects) {
     boolean allowDelete = !isReadOnly && numSelectedProjects > 0;
     boolean allowExport = numSelectedProjects > 0;
     boolean allowExportAll = numProjects > 0;
+    fileDropDown.setItemEnabled(MESSAGES.trashProjectMenuItem(), allowDelete);
```

Comment:
At some point I think we will want to have this menu item say "Delete from Trash" when in in the Trash view. Right now the only way to delete from the trash is to use the button as the menu item is completely neutered in this view.

Reply:
That makes sense. I'll put in an issue for it.